### PR TITLE
Add Q type checks in decomp.cpp and jswalk.c

### PR DESCRIPTION
**Pull request by ehrenjulzert**
For https://github.com/eclipse-openj9/openj9/issues/15861

Signed-off-by: Ehren Julien-Neitzert <ehren.julien-neitzert@ibm.com>

### CODE REVIEW

File: runtime/codert_vm/jswalk.c [hunk 1]
```diff
@@ -1225,6 +1225,7 @@ static void jitWalkResolveMethodFrame(J9StackWalkState *walkState)
 		while ((sigChar = jitNextSigChar(&sigData)) != ')') {
 
 			switch (sigChar) {
+				case 'Q': /* fall through */
```
Comment (hangshao0) [inline runtime/codert_vm/jswalk.c:1228]: You may want to fix the same code in `JITStackWalker.java` as well.

Comment (ehrenjulzert) [reply runtime/codert_vm/jswalk.c:1228]: updated in 888e0d0

**Review by hangshao0 — APPROVED** [review]
LGTM.

**Comment by ehrenjulzert** [comment]
I just pushed an update to add `#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)` guards since I realized I forgot them

**Review by tajila — APPROVED** [review]

**Comment by tajila** [comment]
Jenkins test sanity,extended xlinuxval jdknext

**Comment by tajila** [comment]
Jenkins test sanity win jdk8

**Comment by tajila** [comment]
Jenkins test sanity zlinuxvalst jdknext

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/JITStackWalker.java [hunk 2]
```diff
@@ -555,6 +555,9 @@ private void jitWalkResolveMethodFrame(WalkState walkState) throws CorruptDataEx
 					while ((sigChar = jitNextSigChar(signatureString)) != ')') {
 						
 						switch (sigChar) {
+							/*[IF INLINE-TYPES]*/
```
Comment (tajila) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/JITStackWalker.java:558]: Does DDR code, recognize JPP? @JasonFengJ9

Comment (JasonFengJ9) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/JITStackWalker.java:558]: @tajila yes, an example - https://github.com/eclipse-openj9/openj9/blob/e2890e9141f8b27ab7883a5a4b30413c8c7ececf/debugtools/DDR_VM/src/com/ibm/j9ddr/corereaders/tdump/TDumpReader.java#L119-L121

Comment (tajila) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/JITStackWalker.java:558]: thanks

**Pull request merged**